### PR TITLE
Yoels code review

### PR DESCRIPTION
**Pull request by YoelShoshan**
Few code review and redesign of dl sub-package.
PR isn't ready! 
There is a test that hangs on multiprocessing (doesn't seem related so far to this PR code)

### CODE REVIEW

**Comment by YoelShoshan** [comment]
all tests pass - ready for review

File: fuse/dl/models/model_wrapper.py [hunk 1]
```diff
@@ -31,10 +31,10 @@ class ModelWrapper(torch.nn.Module):
 
     def __init__(self,
                  model: torch.nn.Module = None,
-                 model_inputs: Sequence[str] = ('data.input.input_0.tensor',),
+                 model_inputs: Sequence[str] = None, #('data.input.input_0.tensor',),
```
Comment (mosheraboh) [inline fuse/dl/models/model_wrapper.py:34]: Lets remove the default value.
Move the outputs to be the second argument (and also remove the default value)

File: fuse/dl/models/model_wrapper.py [hunk 2]
```diff
@@ -31,10 +31,10 @@ class ModelWrapper(torch.nn.Module):
 
     def __init__(self,
```
Comment (mosheraboh) [inline fuse/dl/models/model_wrapper.py:32]: Maybe we should follow the dataset naming (ModelWrapSeqToDict)?

**Review by mosheraboh — DISMISSED** [review]

File: fuse/dl/models/model_wrapper.py (hunk 1 above)
Comment (YoelShoshan) [reply fuse/dl/models/model_wrapper.py:34]: done (I assume you meant 3rd argument, write here if you prefer output to be before input)

Comment (YoelShoshan) [reply fuse/dl/models/model_wrapper.py:34]: also prevent positional args, and added example values to the constructor docstring

File: fuse/dl/models/model_wrapper.py (hunk 2 above)
Comment (YoelShoshan) [reply fuse/dl/models/model_wrapper.py:32]: renamed to ModelWrapSeqToDict - did not change the file name as we might include additional wrappers there in the future

File: fuse/dl/models/heads/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from .common import ClassifierFCN, ClassifierFCN3D, ClassifierMLP
+from .head_1d_classifier import Head1DClassifier
```
Comment (mosheraboh) [inline fuse/dl/models/heads/__init__.py:2]: You an issue here caught in unittests with upper/smaller case in 1D/1d

**Review by mosheraboh — APPROVED** [review]
Seems good,
You just need to fix an issue caught in unittests, see the inline comment.

**Pull request merged**